src/types/line.js

```diff
@@ -107,15 +107,28 @@ export default class LineAnnotation extends Element {
 
   draw(ctx) {
     const {x, y, x2, y2, options} = this;
+
     ctx.save();
+    const stroke = setBorderStyle(ctx, options);
+    // no border width, then line is not drawn
+    if (!stroke) {
+      ctx.restore();
+      return;
+    }
```

Comment:
The `stroke` variable is used only for the if, so this block can be condensed.

couple of different styles to do the same thing (saving couple of bytes in size):

```js
    if (!setBorderStyle(ctx, options)) {
      // no border width, then line is not drawn
      return ctx.restore();
    }
```

OR

```js
    if (setBorderStyle(ctx, options)) {
      // the drawing stuff
    }
    ctx.restore();
```

Reply:
done (options 1)